hydrate/component.py

```diff
@@ -66,6 +66,14 @@ def delete_none_attrs(self):
                 delattr(self, key)
 
 
+NO_MATCH_STR = '''No Match Deployments
```

Comment:
Any thoughts of putting all these string constants into single util file?

Reply:
Good idea! Changes reflected in the latest commit.